content/technical/open-source-as-a-service.md

```diff
@@ -0,0 +1,153 @@
+Title: Your Service is not Open Source
+Date: 2020-02-07 18:35
+Tags: open source, technical
+Slug: open-source-as-a-service
+Category: Technical
+Summary: Sharing the source code to your SaaS is insufficient. We effectively enable users to contribute to the a running service for it to be Open Source.
+
+Sharing the source code to your SaaS is insufficient to make it
+actually be Open Source. Sounds self-contradictory?
+
+Most services that espouse "Open Source", do so by simply
+[throwing the code over the wall](https://github.com/travis-ci/travis-ci).
+It's better than nothing, but really misses the point that powers
+Open Source: enabling users to make a change to the software
+they're using.
+
+Some other Open Source services, such as
```

Comment:
I would change "Some other Open Source services" to save "Open Source Services" as the term to use only for our end game, not for what (gitlab|elastic) currently do. 

"Some other popular services powered by Open Source Software" 

(even though this might now be controversial in elastic's case...)

Reply:
agreed. Coining a term is powerful

Reply:
Agree with coining the term. But I wouldn't like to apply the term to these projects (GitLab is Open Core + Managed Service and ElasticSearch has a dubious license now).

Reply:
Updated to use the text @gsalis suggested in quotes.